parts/linux/cloud-init/artifacts/components.json

```diff
@@ -532,28 +532,28 @@
       "multiArchVersionsV2": [
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=oss/kubernetes/kube-proxy",
-          "latestVersion": "v1.27.16",
-          "previousLatestVersion": "v1.27.15"
+          "latestVersion": "v1.27.100-akslts",
```

Comment:
@Devinwong  this will probably break renovate...

Reply:
Just checked in my fork. Fortunately it's smart enough to update it.
![image](https://github.com/user-attachments/assets/c1bb614f-4d25-4d43-a6ee-63dcdf54476a)